.github/workflows/nodejs.yml

```diff
@@ -50,6 +50,10 @@ jobs:
         run: yarn test-ts --selectProjects type-tests
       - name: verify TypeScript@4.3 compatibility
         run: yarn verify-old-ts
+      - name: typecheck examples
```

Comment:
At first I though to run these before type tests. The workflow is stopped after first fail. Feels like feedback from type tests could be more useful. Actually that is not so important, because it is possible to run any of these checks locally.